src/main/scala/io/sdkman/changelogs/java/AzulZuluMigrations.scala

```diff
@@ -2,8 +2,47 @@ package io.sdkman.changelogs.java
 
 import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
 import com.mongodb.client.MongoDatabase
+import io.sdkman.changelogs.{Linux64, MacOSX, Version, Windows, Zulu, removeVersion}
 
 @ChangeLog(order = "019")
 class AzulZuluMigrations {
 
+  @ChangeSet(order = "001", id = "001-add_zulu_7_0_242", author = "jorsol")
+  def migrate001(implicit db: MongoDatabase) = {
+    List(
+      Version("java", "7.0.242-zulu", "https://cdn.azul.com/zulu/bin/zulu7.34.0.5-ca-jdk7.0.242-linux_x64.tar.gz", Linux64, Some(Zulu)),
+      Version("java", "7.0.242-zulu", "https://cdn.azul.com/zulu/bin/zulu7.34.0.5-ca-jdk7.0.242-win_x64.zip", Windows, Some(Zulu))
+    ).validate().insert()
+    Seq(Linux64, Windows, MacOSX).foreach(platform => removeVersion(candidate = "java", version = "7.0.232-zulu", platform))
```

Comment:
This version is no longer in the database. If you are trying to perform deletion of the superseded `7.0.242-zulu` with a patched version, move this statement above the inserts and then try inserting your new versions after that.